### PR TITLE
fix direction calculation

### LINKED ISSUE #51: Directions are sometimes off
The direction arrow sometimes points diagonally even if a horizontal or vertical direction would be much more appropriate.

For example: Sweden is considered to be NE of Algeria, while the bearing is just 15 degrees, which corresponds to N.

Screenshot 2022-02-21 at 22 03 04

### PR DESCRIPTION
**Pull request by friedrich**
fixes #51

### CODE REVIEW

File: src/components/Game.tsx [hunk 1]
```diff
@@ -72,10 +73,12 @@ export function Game({ settingsData }: GameProps) {
         return;
       }
 
+      const bearing = geolib.getGreatCircleBearing(guessedCountry, country);
```
Comment (aplaice) [inline src/components/Game.tsx:76]: I think that the Rhumb-line bearing should give more intuitive results. 

For instance, imagine two points that are at the same latitude, but at very different longitudes — the great circle joining them will go through one of the poles, depending on hemisphere, which isn't what you'd expect.

Also, the rhumb-line bearing, unlike the great-circle bearing, is "symmetrical" (as the bearing is the same throughout the rhumb-line).

(Sorry for randomly commenting on somebody else's PR in somebody else's repo, but I had been just about to also report the "direction" issue and suggest a solution to it, along the lines of your PR. :))

File: src/components/Game.tsx [hunk 2]
```diff
@@ -72,10 +73,12 @@ export function Game({ settingsData }: GameProps) {
...
+
       const newGuess = {
         name: currentGuess,
         distance: geolib.getDistance(guessedCountry, country),
-        direction: geolib.getCompassDirection(guessedCountry, country),
+        direction: getCompassDirection(bearing),
       };
```
Comment (teuteuf) [inline src/components/Game.tsx:82]: I think I would fix it with something like this:
```
      const newGuess = {
        name: currentGuess,
        distance: geolib.getDistance(guessedCountry, country),
        direction: geolib.getCompassDirection(
          guessedCountry,
          country,
          (origin, dest) =>
            Math.round(geolib.getRhumbLineBearing(origin, dest) / 45) * 45
        ),
      };
```

Comment (teuteuf) [reply src/components/Game.tsx:82]: What do you think about it?

Comment (friedrich) [reply src/components/Game.tsx:82]: Looks good! 👍🏻

**Comment by friedrich** [comment]
Closed in favor of @teuteuf's solution.

**Pull request closed without merging**